### DISCUSSION TITLE
Old issues related to #491 "Default values for estate list".

### DISCUSSION THREAD

**Discussion by dai-eastgate** [onOffice-Web-Org/oo-wp-plugin#545] [General]
In issue [#491](https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/491), we have 2 issues in discussion:

1. Field "country" has been renamed to "land".
See the attached file.
![image](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/106214469/9296d721-5be7-4bc5-8759-57a01e9e5aac)

2. In the Form, when we add a new field to the "filter-fields-list".
 "Default Value" of all fields has type="number" input.
See the attached file.
![image](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/106214469/b04f949c-d5fc-4740-82e4-c281b40b89e5)

Would you like us to fix it in the next ramp-up, if not then let us know your plan for these 2 issues. Thanks!

**Comment by dai-eastgate** [comment]
@fredericalpers Please let us know your opinion about this. Thanks!

**Reply by onOffice-Web** [reply]
> @fredericalpers Please let us know your opinion about this. Thanks!

Please go ahead and try to investigate and fix it during ramp up. Otherwise I will postpone it into the next cycle.
